### DISCUSSION TITLE
Microsoft Authentication login

### DISCUSSION THREAD

**Discussion by SebastianBilek** [ArchieHickmott/quotebook#14] [Ideas]
This is probably a must if we want to serve the website on the internet as we don't want randos on the website + we might want to blame people for any bad content.

I found a [cool video](https://youtu.be/oW1SJxGiaZA) that shows of how to get it working with flask, it seems very simple and the guy uses a code example straight from an Azure quickstart. This discussion is here just because, I haven't looked into it any deeper.

**Comment by Helios-fr** [comment]
I would recommend using Firebase for this, it supports Microsoft based authentication, and is also completely free below the limit or 50k monthly active users, which is perfect for our requirements. I also have experience using it.

https://firebase.google.com/
